### PR TITLE
Tip for Encryption

### PR DESCRIPTION
**Pull request by repos-jmp**
The encrypted output is right adjacent to the current speed value and causes confusion, added a tip for newcomers.  [Issue](https://github.com/spring-cloud/spring-cloud-config/issues/561#issuecomment-311092197)

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@repos-jmp Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=730)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=730) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: docs/src/main/asciidoc/spring-cloud-config.adoc [hunk 1]
```diff
@@ -818,6 +818,12 @@ $ curl localhost:8888/encrypt -d mysecret
 NOTE: If the value you are encrypting has characters in it that need to be URL encoded you should use
 the `--data-urlencode` option to `curl` to make sure they are encoded properly.
 
+TIP: The encrypted output when shown on console is obfuscated by adjacent 'Current Speed' value, redirecting the output to file helps avoid this confusion.
```
Comment (ryanjbaxter) [inline docs/src/main/asciidoc/spring-cloud-config.adoc]: Maybe we should make this more general, specifically calling out Current Speed might confuse people.  I think if you just said something like 
"Be sure not to include any of the `curl` command statistics in the encrypted value. 
 Outputting the value to a file can help avoid this problem." 
that would suffice.

**Comment by ryanjbaxter** [comment]
Could you submit this PR against the `1.3.x` branch so we can easily merge it forward?

**Comment by pivotal-issuemaster** [comment]
@repos-jmp Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=730)!

**Comment by repos-jmp** [comment]
Will create a new pull request.

**Pull request closed without merging**